app/uk/gov/hmrc/economiccrimelevystubs/models/des/ReferencesForRemoval.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevystubs.models.des
+
+import play.api.libs.json.{Json, OFormat}
+final case class ReferencesForRemoval(references: List[String])
+
+object ReferencesForRemoval {
+  implicit val format: OFormat[ReferencesForRemoval] = Json.format[ReferencesForRemoval]
+}
```

Comment:
Is there any particular reason for this class? You can just return a `List[String]` serialized as JSON, any need for the wrapper class?

Reply:
I'm used to using wrapper class, but I will remove it, It doesn't serve some special purpose.

Reply:
Removed and fixed tests.